lms/djangoapps/grades/models.py

```diff
@@ -279,7 +287,7 @@ class Meta(object):
     # timestamp for the learner's first attempt at content in
     # this subsection. If null, indicates no attempt
     # has yet been made.
-    first_attempted = models.DateTimeField(null=True, blank=True)
+    first_attempted = models.DateTimeField(null=True, blank=True, db_index=True)
```

Comment:
Wouldn't we want the index to be:
```
(first_attempted, course_id, user_id)
```
?

As we'll want to query by course-wide attempts by all users
And also by course-wide attempts for a single user